src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet("/cars/add")
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/addCar.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Car car = new Car();
+        car.setManufacturer(manufacturerService
+                .get(Long.parseLong(req.getParameter("manufacturerId"))));
+        car.setModel(req.getParameter("model"));
+        car.setDrivers(new ArrayList<>());
+        carService.create(car);
+        resp.sendRedirect(req.getContextPath() + "/index");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/cars");
```

Let's redirect to the corresponding entity page after it's creation. Also, did you forget to include `GetAllCarsController`?